832.py

```diff
@@ -0,0 +1,12 @@
+from typing import List
+
+class Solution:
+    def flipAndInvertImage(self, image: List[List[int]]) -> List[List[int]]:
```

Comment:
Similar to the other problem, we should allocate new space for the resulting image, not modifying the input `image`. If the problem asked you to do everything in-place and not to return anything, then your implementation would be good.